### PR TITLE
Added support for aes-ctr-256

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

**Comment by khenderick** [comment]
Work in progress for #270

File: ovs/lib/albapreset.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+        compression_options = ['snappy', 'bz2', 'none']
+        if compression not in compression_options:
+            raise ValueError('Invalid compression format specified, please choose from: "{0}"'.format('", "'.join(compression_options)))
+
+        encryption_options = ['aes-cbc-256', 'aes-ctr-256', 'none']
+        if encryption not in encryption_options:
+            raise ValueError('Invalid encryption format specified, please choose from: "{0}"'.format('", "'.join(encryption_options)))
+
+        if fragment_size is not None and (not isinstance(fragment_size, int) or not 0 < fragment_size <= 1024 ** 3):
```
Comment (JeffreyDevloo) [inline ovs/lib/albapreset.py]: The last time I polled with Alba they said that the lowest fragment size is 16 Bytes.

File: ovs/lib/albapreset.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+        Deletes a preset from the Alba backend
+        :param alba_backend_guid: Guid of the ALBA backend
+        :type alba_backend_guid: str
+        :param name: Name of the preset
+        :type name: str
+        :return: None
+        """
+        alba_backend = AlbaBackend(alba_backend_guid)
+        preset_default_map = dict((preset['name'], preset['is_default']) for preset in alba_backend.presets)
+        if name not in preset_default_map:
```
Comment (JeffreyDevloo) [inline ovs/lib/albapreset.py]: Missing # validations here?

**Comment by kvanhijf** [comment]
openvstorage-backend-1.7.8-rev.873.16660bb

**Pull request merged**